modules/src/ping.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+
+from templates.text import TextTemplate
+
+def process(input, entitites):
+    output = {}
+    try:
+        url = entities['url'][0]['value']
+        orig_url = url
+        strip_list = ['http://','https://','/']
+        for item in strip_list:
+            url = url.strip(item)
+        page = requests.get('https://isitup.org/'+url+'.json')
+        page = json.loads(page.text)
+        status = page['status_code']
+        if status == 1:
+            txt = orig_url + ' is up!'
+        elif status == 2:
+            txt = orig_url + ' seems to be down!'
+        else :
```

Comment:
Do we know the status code here? If yes, then we should use that here instead. Any unexpected status codes should result in an error.